Libplanet/Tx/RawTransaction.cs

```diff
@@ -70,26 +53,26 @@ ImmutableArray<byte> signature
             Signature = signature;
         }
 
-        public RawTransaction(Dictionary<string, object> dict)
+        public RawTransaction(IValue value)
```

Comment:
How about restricting the type of `value` here to `Dictionary`?

Reply:
If so, should we restrict return type of `ToBencodex()` to `Bencodex.Types.Dictionary` either?

Reply:
Maybe yes. if we were implementing for other types than `Transaction<T>` or `Block<T>`, it can be quite weird that `ToBencodex()` returns `Dictionary` instead of `IValue`. but in this case, it seems that we only implement `ToBencodex()` for `Transaction<T>` and `Block<T>` and they can be treat as complicated composite types. so IMHO it's okay that we also restrict `ToBencodex`'s return type to `Dictionary`.

Reply:
Great, I've applied your suggestions :)